pkg/apis/policy/types.go

```diff
@@ -389,6 +392,9 @@ type SupplementalGroupsStrategyOptions struct {
 type SupplementalGroupsStrategyType string
 
 const (
+	// SupplementalGroupsStrategyMayRunAs means that container does not need to run as a particular gid.
```

Comment:
s/to run as a particular gid/to run with particular gids/ ?

Reply:
It was but a mere copy-paste but "with" sounds definitely better